lib/src/components/NxCollapsibleItemsSelect/commonTypes.ts

```diff
@@ -4,15 +4,14 @@
  * the terms of the Eclipse Public License 2.0 which accompanies this
  * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
  */
-import {ReactNode} from 'react';
+import {ReactNode, ReactElement} from 'react';
 import * as PropTypes from 'prop-types';
 
 import { requiredNullableString } from '../../util/customPropTypes';
-import RequiredReactNode from '../../util/RequiredReactNode';
 
 export interface Option {
   id: string | null;
-  name: RequiredReactNode;
+  name: string | ReactElement;
```

Comment:
Since the initial ticket https://sonatype.atlassian.net/browse/RSC-1283 was looking for IQ to have the ability to add in JSX as an option.name, what are your thoughts of changing the `name` type to `string | ReactElement` instead of `ReactNode`? (since we won't need number, boolean etc). This cleans up the casting of `option.any` required wherever we need to use the `Option` interface with the name typed as `RequiredReactNode`

Reply:
If this is an acceptable route forward, I'd need to make a slight adjustment to the gallery docs as well